pkgs/by-name/li/livi/package.nix

```diff
@@ -0,0 +1,54 @@
+{ lib
+, stdenv
+, fetchFromGitLab
+, meson
+, ninja
+, pkg-config
+, gst_all_1
+, wrapGAppsHook4
+, appstream-glib
+, gtk4
+, libadwaita
+, desktop-file-utils
+, libGL
+}:
+
+stdenv.mkDerivation rec {
+  pname = "livi";
+  version = "0.0.6";
+
+  src = fetchFromGitLab {
+    owner = "guidog";
+    repo = "livi";
+    domain = "gitlab.gnome.org";
+    rev = "v${version}";
+    sha256 = "sha256-DaIbBCJT4Da5noW6Q5z1yzTZ256HNqrvdXgwSY7p/D8=";
+  };
+  nativeBuildInputs = [
+    meson
+    ninja
+    pkg-config
+    wrapGAppsHook4
+    gst_all_1.gstreamer
+    gst_all_1.gst-plugins-base
+    gst_all_1.gst-plugins-bad
+    gst_all_1.gst-plugins-good
+    appstream-glib
+    gtk4
+    libadwaita
+    desktop-file-utils
+  ];
+
+  buildInputs = [
+    libGL
+  ];
+
+  meta = with lib; {
+    homepage = "https://gitlab.gnome.org/guidog/livi";
+    description = "A small video player targeting mobile devices (also named μPlayer)";
+    license = licenses.gpl3;
```

Comment:
```suggestion
    license = licenses.gpl3Plus;
```

Reply:
> 

I think it should be changed into `gpl3Only` based on this https://gitlab.gnome.org/guidog/livi/-/blob/main/COPYING
I seems like `gpl3` is deprecated but I didn't get any warnings in the build. 
Do I know why my builds don't care about the meta section? It was the same with the maintainer field as well. I didn't notice any issues with it but Borg caught it and failed.

Reply:
> I think it should be changed into gpl3Only based on this https://gitlab.gnome.org/guidog/livi/-/blob/main/COPYING

It's at the top of c source files. Also the `flathub` uses `gpl3Plus`.

> It seems like gpl3 is deprecated but I didn't get any warnings in the build.
Do you know why my builds don't care about the meta section? It was the same with the maintainer field as well. I didn't notice any issues with it but Borg caught it and failed.

Because It does not use these sections.